autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,135 @@
+fi
+
+PARTITIONS="${DISKSLICE} gpt"
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz lib32.txz src.txz ports.txz"
+# DISTRIBUTIONS="kernel.txz base.txz"
+
+
+#!/bin/sh
+set -o xtrace
+
+echo 'Running installerconfig ...' > /dev/ttyu0
+# Enable serial and internal consoles
+echo "Enabling serial and internal consoles" 
+echo "-Dhv" > /boot.config
+echo "cuau0   \"/usr/libexec/getty std.9600\"  xterm   on  secure" >> /etc/ttys
+
+#hostname
+sysrc hostname="FreeBSD"
+
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 da0
+# Set Time Zone to UTC
+echo "Setting Time Zone to UTC ..."
+/bin/cp /usr/share/zoneinfo/UTC /etc/localtime
+/usr/bin/touch /etc/wall_cmos_clock
+/sbin/adjkerntz -a
+
+echo "Set DHCP to NIC ${ifdev} ..." > /dev/ttyu0
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}"
+sysrc ifconfig_${ifdev}=DHCP
+sleep 5
+
+# Set latest repository to the new system
+echo "Set latest repository to the new system ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -f | cat
+bit=`uname -m`
+OSFamily=`uname -s`
+major_version=`freebsd-version -k | cut -d . -f 1`
+repo="${OSFamily}:${major_version}:${bit}"
+
+mkdir -p /usr/local/etc/pkg/repos
+cat << EOF > /usr/local/etc/pkg/repos/latest.conf
+FreeBSD: { enabled: no }
+
+latest: {                                                                     
+  url: "pkg+http://pkg.FreeBSD.org/${repo}/latest",
+  mirror_type: "srv",                                                           
+  signature_type: "fingerprints",                                               
+  fingerprints: "/usr/share/keys/pkg",                                         
+  enabled: yes                                                                 
+}
+EOF
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
+
+# Installing packages
+echo "Installing packages ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -y
+env ASSUME_ALWAYS_YES=YES pkg update -f
+# Hit issue: reset by peer during install packages
+# The open-vm-tools is not installed by default
+packages_to_install='python3 open-vm-tools-nox11 sudo'
+for package_to_install in $packages_to_install
+do
+    echo aaa | grep bbb
+    until [ $? -eq 0 ]
+    do
+        echo "will install package: $package_to_install" > /dev/ttyu0
+        check_results=`env ASSUME_ALWAYS_YES=YES pkg install -y $package_to_install`
+        echo $check_results > /dev/ttyu0
+        echo $check_results | grep "are already installed"
+    done
+    echo "The package($package_to_install) is already installed" > /dev/ttyu0
+done
+# env ASSUME_ALWAYS_YES=YES pkg install -y sudo lsblk bash curl wget sudo
+
+# Add new user. 
+{% if new_user is defined and new_user != 'root' %}
+echo "{{ vm_password }}" | pw -V /etc useradd {{ new_user }} -h 0 -s /bin/sh -G wheel -d /home/{{ new_user }} -c "{{ new_user }} User"
+mkdir -p /home/{{ new_user }}
```

Comment:
useradd doesn't create home directory automaticall.

Thanks

Reply:
I saw below option in https://man.freebsd.org/cgi/man.cgi?pw. Does the option `-m` work?
```
 -m		   This	option instructs pw to attempt to create the user's
		   home	directory.  While primarily useful when	adding a new
		   account with	useradd, this may also be of use when moving
		   an existing user's home directory elsewhere on the file
		   system.  The	new home directory is populated	with the con-
		   tents of the	skeleton directory, which typically contains a
		   set of shell	configuration files that the user may person-
		   alize to taste.  Files in this directory are	usually	named
		   dot.<config>	where the dot prefix will be stripped.	When
		   -m is used on an account with usermod, existing configura-
		   tion	files in the user's home directory are not overwritten
		   from	the skeleton files.
```

Reply:
tried and seems the option "-m" does not work.
![image](https://user-images.githubusercontent.com/41054978/222403677-107a57bf-bd70-4776-b68d-315bb0fac2c0.png)

Reply:
I saw it works
<img width="718" alt="image" src="https://user-images.githubusercontent.com/5078633/222413894-ca0183e1-010e-46d7-894f-2176881884b6.png">